tf2_ros/include/tf2_ros/buffer_client.h

```diff
@@ -132,7 +132,6 @@ namespace tf2_ros
       geometry_msgs::TransformStamped processGoal(const tf2_msgs::LookupTransformGoal& goal) const;
       geometry_msgs::TransformStamped processResult(const tf2_msgs::LookupTransformResult& result) const;
       mutable LookupActionClient client_;
-      double check_frequency_;
```

Comment:
If we leave this here with a note that it's unused then this change can be ABI compatible and can then be made available in active distros without need for a discussion.